doc/md/guides/quickstart.mdx

```diff
@@ -114,9 +134,14 @@ Run the rest of the commands in this guide from the root of the repository.
 The Bank of Holos platform is organized as a collection of software components.
 Each component represents a piece of software provided by an upstream vendor,
 for example ArgoCD, or software developed in-house.  Components are also used to
-glue together, or integrate, other components into the platform.
+glue together, or integrate, other components into the platform. ArgoCD is a good
+first example for a platform because:
+
+  - It's a component that's easily deployed via an upstream Helm chart.
+  - It contains a resource (`AppProjects`) that's used by other components to create ArgoCD Application definitions.
```

Comment:
"It" is referring to ArgoCD, but the implementation is multiple components.  There's 2 components just to manage ArgoCD itself, then there's the `app-projects` component which is the thing that manages AppProjects.  Then there's the [ArgoConfig](https://holos.run/docs/api/author/v1alpha3/#ArgoConfig) that mixes in Application resources to each component in the platform.